assets/src/wizards/subscriptions/index.js

```diff
@@ -92,6 +93,15 @@ class SubscriptionsWizard extends Component {
 						disabled
 					/>
 				</Card>
+				<Card>
+					<FormattedHeader
+						headerText="Progress bar"
+					/>
+					<ProgressBar completed="2" total="3" />
+					<ProgressBar completed="2" total="5" label="Progress made" />
+					<ProgressBar completed="0" total="5" displayFraction />
```

Comment:
Not suggesting this be committed, but as a way to test this component I set the `completed` attribute to `{ inputTextValue2 }`, which lets me change progress in real time by changing the value of the second text input.